apps/strapi-cms/src/api/webinar/content-types/webinar/lifecycles.ts

```diff
@@ -20,12 +18,14 @@ const validateDates = (event: IWebinarEvent) => {
   const endDateTime = data.endDatetime ? new Date(data.endDatetime) : null;
 
   if ((startDateTime && !endDateTime) || (!startDateTime && endDateTime)) {
+    // eslint-disable-next-line functional/no-throw-statements
```

Comment:
**question**: why this disable?

Why this rule has been disabled into a function and not listed into the `eslint-strapi` rules?

Reply:
I added this rule to excluded rules. At start i thought that it could be possible to manage error without throwing error but it is only possible in controller and not in lifecycle hook.